pkgs/by-name/zo/zotero-beta/package.nix

```
@@ -2,104 +2,78 @@
 , stdenv
 , fetchurl
 , wrapGAppsHook3
-, autoPatchelfHook
 , makeDesktopItem
+, alsa-lib
 , atk
 , cairo
 , coreutils
-, curl
-, cups
-, dbus-glib
 , dbus
+, dbus-glib
 , dconf
-, fontconfig
-, freetype
 , gdk-pixbuf
 , glib
-, glibc
+, gnome
+, gsettings-desktop-schemas
 , gtk3
+, libGL
 , libX11
-, libXScrnSaver
-, libxcb
 , libXcomposite
 , libXcursor
 , libXdamage
 , libXext
 , libXfixes
 , libXi
-, libXinerama
-, libXrender
+, libXrandr
 , libXt
-, libnotify
-, gnome
-, libGLU
-, libGL
-, nspr
-, nss
-, pango
-, gsettings-desktop-schemas
-, alsa-lib
 , libXtst
+, libxcb
+, mesa
+, pango
+, pciutils
 }:
 
 stdenv.mkDerivation rec {
   pname = "zotero";
-  version = "7.0.0-beta";
+  version = "7.0.0-beta.79+95d5a5417";
+  safeVersion = builtins.replaceStrings ["+"] ["%2B"] version;
 
   src = fetchurl {
-    url = "https://download.zotero.org/client/beta/${version}.65%2Bb047f3d90/Zotero-${version}.65%2Bb047f3d90_linux-x86_64.tar.bz2";
-    hash = "sha256-e7T/hvb8bFb4hkouihBkgqUu4ugYoUMHNVKxh0r/WUM=";
+    url = "https://download.zotero.org/client/beta/${safeVersion}/Zotero-${safeVersion}_linux-x86_64.tar.bz2";
```

Don't pass this to `mkDerivation` directly. A let-in would be much more appropriate here
```suggestion
  src = let 
    escapedVersion = builtins.replaceStrings ["+"] ["%2B"] version;
  in
  fetchurl {
    url = "https://download.zotero.org/client/beta/${safeVersion}/Zotero-${escapedVersion}_linux-x86_64.tar.bz2";
```